addOns/fuzz/src/main/java/org/zaproxy/zap/extension/fuzz/payloads/ui/impl/NumberPayloadGeneratorAdapterUIHandler.java

```diff
@@ -267,7 +267,7 @@ public void clear() {
             getPayloadsPreviewTextArea().setText("");
             fromField.setValue(0);
             toField.setValue(0);
-            stepField.setValue(0);
+            stepField.setValue(1);
```

Comment:
Corresponding to:
https://github.com/zaproxy/zap-extensions/blob/b84afa93200685de975173a2c9dc2c53a9fa7f1f/addOns/fuzz/src/main/java/org/zaproxy/zap/extension/fuzz/payloads/ui/impl/NumberPayloadGeneratorAdapterUIHandler.java#L169

Reply:
Worth extracting a constant.